.github/workflows/hrm-lambda-release.yml

```diff
@@ -0,0 +1,113 @@
+name: 'Deploy HRM Lambda'
+on:
+  workflow_dispatch:
+    inputs:
+      lambda_name:
+        description: 'hrm lambda name'
+        required: true
+        default: retrieve-recording-url
+        type: choice
+        options:
+          - retrieve-recording-url
+          - retrieve-transcript
+
+      environment:
+        description: HRM environment to deploy. E.G = development, staging, production (must match with the AWS environment value). Default value = development
+        default: development
+        required: true
+        type: choice
+        options:
+          - development
+          - staging
+          - production
+
+      send-slack-message:
+        description: 'Specifies if should send a Slack message at the end of successful run. Defaults to true'
+        required: true
+        type: boolean
+        default: true
+
+  workflow_call:
+    secrets:
+      AWS_ACCESS_KEY_ID:
+        required: true
+      AWS_SECRET_ACCESS_KEY:
+        required: true
+      # Should probably use the passed in region for everything but the GITHUB_ACTIONS_SLACK_BOT_TOKEN SSM parameter only exists in us-east-1
+      AWS_DEFAULT_REGION:
+        required: true
+
+env:
+  GITHUB_SHA: ${{ github.sha }}
+
+jobs:
+  build-lambda:
+    name: Build And Deploy Lambda
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.x
+
+      #TODO: Move to a matrix based region split for the ECR/update lambda steps
+      - name: Set Env Based on Environment
+        # this is overcomplicated to allow for testing with a push event
```

Comment:
It could become less complicated since push triggers are disabled, but for simplicity of testing, I'd like to leave it in until things are a bit more settled.